tests/unit/test_schemas.py

```diff
@@ -81,7 +81,7 @@ def validate_node(tree, node, schema, label, name):
 {textwrap.indent(str(error), '  ')}
 """)
 
-        assert False, f'{label} {name} fails validation'
+        raise AssertionError(f"{label} {name} fails validation")
```

Comment:
This one should probably be `pytest.fail` too.

Reply:
Make sense, thanks.